pymeshfix/cython/_meshfix.pyx

```diff
@@ -197,12 +197,12 @@ cdef class PyTMesh:
             raise Exception('Cannot load new arrays once initialized')
 
         if not v.flags['C_CONTIGUOUS']:
-            if v.dtype != np.float:
-                v = np.ascontiguousarray(v, dtype=np.float)
+            if v.dtype not in (float, np.float64):
+                v = np.ascontiguousarray(v, dtype=np.float64)
             else:
                 v = np.ascontiguousarray(v)
-        elif v.dtype != np.float:
-            v = v.astype(np.float)
+        elif v.dtype not in (float, np.float64):
```

Comment:
`v.dtype == float` can never be true, right? Either it's a numpy floating type or `object`.

In fact this snippet always converts to contiguous double, right?
```py
>>> arr = np.arange(3.)

>>> np.ascontiguousarray(arr, dtype=np.float_) is arr
True
```

Unless I'm missing some edge case this _whole_ if could be replaced with this one call. If the array is already C-contiguous double, it will be left intact. Otherwise it will be copied to a contiguous double array.